projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -103,6 +103,10 @@ export class AppService implements OnDestroy {
 
     this.authenticationService.onLogout.subscribe(() => {
       searchQueryBuilderService.resetToDefaults();
+      acaMobileAppSwitcherService.clearSessionExpireTime();
+      if (acaMobileAppSwitcherService.dialogRef !== null && acaMobileAppSwitcherService.dialogRef !== undefined) {
```

Comment:
make the dialogRef private and move the check for the "dialogRef" inside the "closeDialog" method

Reply:
done